build/yaml-templates/prepare_templates.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$DEVWORKSPACE_BRANCH" ]; then
+  echo "Environment variable DEVWORKSPACE_BRANCH must be set"
+  exit 1
+fi
+
+BASE_DIR=${1:-/build}
+OUTPUT_DIR="${BASE_DIR}/deploy"
+TARBALL_PATH="${BASE_DIR}/devworkspace_operator_templates.tar.gz"
+
+# Clone repo
+git clone --depth=1 --branch "${DEVWORKSPACE_BRANCH}" https://github.com/devfile/devworkspace-operator.git
+cd devworkspace-operator
+
+# Grab devfile/api CRDs
+./update_devworkspace_crds.sh --init --api-version "$DEVWORKSPACE_API_VERSION"
+
+# Fill env in template files
+mv config/cert-manager/kustomization.yaml config/cert-manager/kustomization.yaml.bak
```

Comment:
That seems to duplicate stuff we have in Makefile, which means we have more to maintain and test.

According to discussion with @tolusha, it would simplify things for chectl if it can just grab static files from devworkspace-operator repo.
After go vendoring we're not afraid of redundant ~16k lines )

So, it make me thinking what if:
1. We have a script that generates static deploy files, according to the current env vars.
2. We generate static deploy files with default values and keep them in the repo, PR checks that they are up to date.
3. Makefile invokes the script from 1, but result is stored in another not commited place.

@amisevsk WDYT?

Reply:
Makes sense; my main concern is the levels of indirection (pkg/api -> CRDs -> deployable yamls) but that can be addressed with PR checks I suppose. I'll close this one and port the changes over to a new one that includes the full yamls.